src/archiefvernietigingscomponent/emails/constants.py

```diff
@@ -7,6 +7,7 @@ class EmailTypeChoices(DjangoChoices):
     report_available = ChoiceItem("report_available", _("Report available"))
     review_required = ChoiceItem("review_required", _("Review required"))
     changes_required = ChoiceItem("changes_required", _("Changes required"))
+    update_required = ChoiceItem("update_required", _("Update required"))
```

Comment:
It is not too clear just by looking at this what the difference between `changes_required` and `update_required` is. Maybe something like `review_reminder` might be clearer.